sirepo/package_data/static/js/radia.js

```diff
@@ -2040,6 +2075,25 @@ SIREPO.app.directive('kickMapReport', function(appState, panelState, plotting, r
     };
 });
 
+SIREPO.app.directive('pointEditor', function(appState, geometry, panelState, plotting, radiaService, utilities) {
+
+    return {
+        restrict: 'A',
+        scope: {
+            modelName: '=',
+            model: '=',
+        },
+        template: `
+            <div>
+                <div data-ng-repeat=""></div>
+            </div>
+        `,
+        controller: function($scope, $element) {
+            //<div data-report-content="parameter" data-model-key="extrudedPoly" data-report-id=""></div>
```

Comment:
Don't completely understand the commented out html

Reply:
[52f0150](https://github.com/radiasoft/sirepo/pull/4686/commits/52f015084417fcf016db76b1785b29e7616c95b7)

It was to have been a mini-plot of the shape, but I reduced the scope of the issue. No need to keep it